first.py

```diff
@@ -0,0 +1,22 @@
+"""Hey this i smy cool module
```

Comment:
remove this file

Reply:
I think I did it right now. Thanks for the comment.